wiki.md

```diff
@@ -14,12 +14,23 @@ you have to install go and set $GOPATH and $GOBIN to ~/.bashrc.
 ```sh
 brew intall golang
 ```
+If you use `Linuxbrew` on Linux, you can install via the above command.  
```

Comment:
```suggestion
If you use [`Linuxbrew`](https://docs.brew.sh/Homebrew-on-Linux) on Linux, you can install via the above command.  
```

Reply:
Maybe adding the ref URL would be nicer :)

Reply:
Thanks!